doc/concepts/pipeline-concepts/pipeline/index.rst

```diff
@@ -0,0 +1,61 @@
+Pipeline
+========
+
+A pipeline is a Pachyderm primitive that is responsible for collecting data
+from a specified source, or ``pfs/input``, transform it according to the pipeline
+configuration file, and place it to the ``pfs/out`` directory in the ``pachd``
+container. Each pipeline subscribes to a branch in the
+input repository or repositories, gets triggered by a commit,
+and runs your code to completion.
+
+A minimum pipeline specification must include the following parameters:
+
+- ``name`` — The name of your data pipeline. You can set an arbitrary
+  name that is meaningful to the code you want to run to process the
+  data in the specified repository.
+- ``input`` — A location of the data that you want to process, such as a
+  Pachyderm repository or a Git repository. You can specify multiple input
+  repositories, as well as combine the repositories as union or cross
```

Comment:
> repositories, as well as combine the repositories as union or cross
  pipelines.
  Depending on the anticipated results, you can unite and cross
  pipelines as needed in this field. For more information, see `Cross
  and Union <cross-union.html>`__.

repositories and have their data combined in a variety of ways and as it's passed to the user code for processing. For more information, see `Cross and Union <cross-union.html>`__.

Note: it'll be cross, union, and join soon. So we'll decide how we want to organize those later.